dokku

```diff
@@ -89,7 +89,11 @@ case "$1" in
         BIND_EXTERNAL=$(pluginhook bind-external-ip $APP)
 
         [[ -n "$DOKKU_BUILDSTEP" ]] && START_CMD="/start $PROC_TYPE"
-        [[ -z "$DOKKU_BUILDSTEP" ]] && DOKKU_DOCKERFILE_PORT=$(dokku config:get $APP DOKKU_DOCKERFILE_PORT || true)
+
+        if [[ -z "$DOKKU_BUILDSTEP" ]];then
+          DOKKU_DOCKERFILE_PORT=$(dokku config:get $APP DOKKU_DOCKERFILE_PORT || true)
+          START_CMD=$(dokku config:get $APP DOKKU_DOCKERFILE_START_CMD)
```

Comment:
This will break if DOKKU_DOCKERFILE_START_CMD is not set. Example:
```
root@dokku:~/dokku# dokku config:get node-js-app DOKKU_DOCKERFILE_START_CMD ; echo $?
1
```